lib/neuron/config.ex

```diff
@@ -34,20 +34,26 @@ defmodule Neuron.Config do
 
       iex> Neuron.Config.set(:process, url: "http://example.com/graph")
       :ok
+
+      iex> Neuron.Config.set(json_response_opts: [keys: :atoms])
+      :ok
   """
 
   @spec set(context :: :global | :process, value :: keyword()) :: :ok
   def set(context, value)
 
   def set(:global, nil) do
-    [:neuron_url, :neuron_headers, :neuron_connection_opts]
+    [:neuron_url, :neuron_headers, :neuron_connection_opts, :neuron_json_response_opts]
     |> Enum.map(&Store.delete(:global, &1))
   end
 
   def set(context, url: value), do: Store.set(context, :neuron_url, value)
   def set(context, headers: value), do: Store.set(context, :neuron_headers, value)
   def set(context, connection_opts: value), do: Store.set(context, :neuron_connection_opts, value)
 
+  def set(context, json_response_opts: value),
```

Comment:
I'd rather call it `parse_options`, is that ok for you?

Reply:
I agree that parse_options is better than json_response_opts. I'll make the change.